tests/Cart/AddToCart.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '../../src/fixtures/UsedFixtures';
+import { CartPage } from '../../src/pages/CartPage';
+import { MenuPage } from '../../src/pages/MenuPage';
+import { NavHeader } from '../../src/components/NavHeader';
+
+let navHeader: NavHeader;
+let menuPage: MenuPage;
+let cartPage: CartPage;
+
+test.describe('Test adding to cart via confirmation dialog', () => {
+  test.beforeEach(async({ page }) => {
+    navHeader = new NavHeader(page);
```

Comment:
This one I proposed previously to add to the BasePage. Then you will be able to write the next code:

menuPage.header.toMenu();

Reply:
moved those navigation methods to BasePage and deleted NavGeader page object